administration/src/errors/getMessageFromApolloError.tsx

```diff
@@ -2,27 +2,22 @@ import { ApolloError } from '@apollo/client'
 import { ReactElement } from 'react'
 
 import defaultErrorMap from './DefaultErrorMap'
+import graphQlErrorMap from './GraphQlErrorMap'
 
-type GraphQLErrorMessage = {
+export type GraphQLErrorMessage = {
   title: string
   description?: string | ReactElement
 }
 
 const getMessageFromApolloError = (error: ApolloError): GraphQLErrorMessage => {
-  const defaultMessage = 'Etwas ist schief gelaufen.'
-
-  if (error.networkError) {
```

Comment:
not sure if everything is still catched. All i tested was catched by `DefaultErrorMap`
Maybe there is also sth misconfigured so that 403 were classified as `NetworkErrors` accidently?